README.md

```diff
@@ -32,6 +32,29 @@ Visit the `examples/` directory here for more information.
 Additional payload applications are maintained in a separate repository that actually integrate payload hardware for
 demonstration purposes: https://github.com/antaris-inc/SatOS-Payload-Demos
 
+### Compile c++ applictaion
+
+* Launch Build Environment
+  `make build_env` will instantiate a build environment container and open a new shell within it.
+
+```
+make build_env
```

Comment:
Is this the only way to compile the cpp application? It might be easier if we could wrap/hide the container invocation from the user (should the user opt for simplicity over control) and have a 'make cpp_example' target that:
- maps the user directory to a container
- invokes g++ inside the container
- drops the errors to the console and the compiled binary in the user's directory on the host.